spec/models/protocol_subscription_spec.rb

```diff
@@ -326,38 +326,48 @@
     it 'should create responses when you create a protocol subscription' do
       protocol = FactoryBot.create(:protocol)
       FactoryBot.create(:measurement, :periodical, protocol: protocol)
+
+      # ANDO: Heb jij enig idee waarom ik deze reloads moet doen ineens?
```

Comment:
zie comment

Reply:
Volgens mij is het niet nodig. Tenzij die Responses.joins(:measurements).find(...) query iets raars veroorzaakt van dattie measurements gaat cachen oid. Maar dat zou niet moeten kunnen.